docs/current/tools.md

```diff
@@ -17,17 +17,78 @@
             </style>
 </head><article class="markdown-body">
 
-# Vaquero Validator
+# Vaquero Tools
 
-[Home](https://ciscocloud.github.io/vaquero-docs/) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master)
+[Home](https://ciscocloud.github.io/vaquero-docs/) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master) | 
+[Migrate](#vaquero-migrate) | [Preview](#vaquero-preview) | [Validator](#vaquero-validator)
 
+## Vaquero Migrate
+The Vaquero migrate tool is built for moving and renaming stored information about SoTs and Sites. 
+By default, the tool will check for collisions in the destination storage. If it finds any, it will 
+ask for the user to confirm to continue and overwrite any information.
+
+### Using vaquero migrate
+
+- Migrating files from one local storage to the destination etcd: 
+`vaquero migrate --src <config_with_local>.yaml --dst <config_with_etcd>.yaml`
+- Migrating and renaming an SoT:
+`vaquero migrate --src <config>.yaml --dst <config>.yaml --src-sot <src_sot_id> --dst-sot <new_id> --del`
+
+Options:
```

Comment:
this is showing up in paragraph form: https://ciscocloud.github.io/vaquero-docs/docs/branches/migrate%2392/tools.html